src/store/mode/types.ts

```diff
@@ -0,0 +1,11 @@
+export enum ModeTypes {
```

Comment:
We usually don't append `Type` to the end of the type name. How about `export enum Modes {`?